File: read.java
import java.io.BufferedReader;
import java.io.FileReader;

class Read {
    public static void main(String[] args) throws Exception {
        String line;
        BufferedReader reader = new BufferedReader(new FileReader("input.txt"));
        while ((line = reader.readLine()) != null)
        {
        System.out.println(line);
        }
    reader.close();
        
    
}
}

File: read.js
const fs = require('fs');
const readline = require('readline');

async function main() {
  const fileStream = fs.createReadStream('input.txt');

  const rl = readline.createInterface({
    input: fileStream,
    crlfDelay: Infinity
  });
  for await (const line of rl) {
    console.log(line);
  }
}

main();